code/game/objects/items/implants/implant_freedom.dm

```diff
@@ -5,35 +5,44 @@
 	implant_color = "r"
 	uses = 4
 
+/obj/item/implant/freedom/implant(mob/living/target, mob/user, silent, force)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!iscarbon(target)) //This is pretty much useless for anyone else since they can't be cuffed
+		balloon_alert(user, "that would be a waste!")
+		return FALSE
+	return TRUE
 
 /obj/item/implant/freedom/activate()
 	. = ..()
+	var/mob/living/carbon/carbon_imp_in = imp_in
+	if(!carbon_imp_in.handcuffed && !carbon_imp_in.legcuffed)
+		balloon_alert(carbon_imp_in, "no restraints!")
+		return
+
 	uses--
-	to_chat(imp_in, span_hear("You feel a faint click."))
-	if(iscarbon(imp_in))
-		var/mob/living/carbon/C_imp_in = imp_in
-		C_imp_in.uncuff()
+
+	balloon_alert(carbon_imp_in, "bindings released!")
+	carbon_imp_in.uncuff()
 	if(!uses)
+		addtimer(CALLBACK(carbon_imp_in, TYPE_PROC_REF(/atom, balloon_alert), carbon_imp_in, "implant degraded!"), 1 SECONDS)
 		qdel(src)
 
-
 /obj/item/implant/freedom/get_data()
 	var/dat = {"
 <b>Implant Specifications:</b><BR>
 <b>Name:</b> Freedom Beacon<BR>
-<b>Life:</b> optimum 5 uses<BR>
+<b>Life:</b> Optimum 4 uses<BR>
```

Comment:
```suggestion
<b>Life:</b> Optimum [initial(uses)] uses<BR>
```
If it was changed from 5 > 4 at some point it could change again. I don't know if this actually works inside this wrapper though.

Reply:
Value prints as expected